handlers/iamInfoHandler.go

```diff
@@ -22,19 +22,19 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/netflix/weep/metadata"
 	"github.com/netflix/weep/util"
 )
 
 func IamInfoHandler(w http.ResponseWriter, r *http.Request) {
-
-	awsArn, _ := util.ArnParse(metadata.Role)
+	// TODO: this was crashing because of a nil pointer dereference. Fix it!
```

Comment:
Not sure if you meant to commit with this TODO. 
Is it `nil` because a role may not be specified for the new combined metadata and ECS mode? would a simple check  to check `metadata.Role` first suffice?

Reply:
This is a TODO for later, along with the others. The old `metadata` package had almost everything refactored out of it other than a few variables that were directly accessed, which was an anti-pattern. I'll be able to fix it with the CLI UX refactor since we'll be able to count on a "default" role being set.

Also, the endpoint that this handler is for doesn't get called frequently (or maybe at all) -- if it was, it would panic with a nil pointer dereference. There's a decent chance we could just remove it entirely.